tests/ember_debug/deprecation-debug-test.js

```diff
@@ -93,8 +105,18 @@ module('Ember Debug - Deprecation', function (hooks) {
       Route.extend({
         setupController() {
           EmberDebug.IGNORE_DEPRECATIONS = false;
-          deprecate('Deprecation 1', false, { id: 'dep-1', until: '1.0.0' });
-          deprecate('Deprecation 2', false, { id: 'dep-2', until: '1.0.0' });
+          deprecate('Deprecation 1', false, {
+            id: 'dep-1',
+            for: 'ember-inspector',
+            since: '0.1.0',
```

Comment:
I thought this has to be an object.. is that not the case? Other than that seems good to me